cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -118,6 +118,9 @@ n.propertyName_n)
 | Drop a composite index without specifying a name. 
 |===
 
+[NOTE]
+The `IF [NOT] EXISTS` syntax for indexes is only available in Neo4j 4.1.3 and onwards.
```

Comment:
Maybe add the note in `create a single property index only if it does not already exist` and `drop a non-existing index` as well?